src/main/java/frc/robot/subsystems/climber/commands/JoystickClimb.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot.subsystems.climber.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.climber.Climber;
+import frc.robot.utils.Utils;
+
+import java.util.function.BooleanSupplier;
+import java.util.function.DoubleSupplier;
+
+/**
+ * parameters: Climber the climber, BooleanSupplier stop, DoubleSupplier joystickOutput.
+ * This command let the driver control the climb movement by moving Xbox joystick.
+ */
+
+public class JoystickClimb extends CommandBase {
+    private final Climber climber;
+    private final BooleanSupplier isFinished;
+    private final DoubleSupplier joystickOutput;
+
+    public JoystickClimb(Climber climber, BooleanSupplier isFinished, DoubleSupplier joystickOutput) {
+        this.climber = climber;
+        this.isFinished = isFinished;
```

Comment:
I saw that the is finished depends on if the y button is pressed. 
You can simplify by changing it from default command to a `whileHeld` command and avoid dealing with the `isFinished`.